recipe/meta.yaml

```diff
@@ -26,8 +26,12 @@ requirements:
     - {{ compiler('cxx') }}
   host:
     - lz4-c 1.9.4
-    - zstd 1.5.4
+    - zstd 1.5.2
```

Comment:
That's curious. Upstream are vendoring 1.5.5, https://github.com/Blosc/c-blosc2/compare/v2.8.0..v2.10.5#diff-68d24fa81558aae3d8c59e2aa57a4fa719ea3b04d7fa14beff45f16f00858f50R140.

Reply:
I approved anyway since `zstd` exports `- {{ pin_subpackage('zstd', max_pin='x.x') }}`.

Reply:
If you are vendoring, it makes sense to use the latest available.